src/utils/localStorage/localStorage.js

```diff
@@ -0,0 +1,4 @@
+export const getFromLocalStorage = key => localStorage.getItem(key);
+export const setToLocalStorage = (key, value) =>
+  localStorage.setItem(key, value);
+export const removeFromLocalStorage = key => localStorage.removeItem(key);
```

Comment:
with names like that, i'm not sure these functions achieve anything. You create a new api with no added functionality and very similar names.

Reply:
jo, good point. issimsiu